src/cow-react/modules/limitOrders/updaters/QuoteUpdater/hooks/useFetchMarketPrice/index.ts

```diff
@@ -34,14 +38,13 @@ export function useFetchMarketPrice() {
         return
       }
 
-      getQuote(feeQuoteParams)
+      getQuoteOnlyResolveLast(feeQuoteParams)
```

Comment:
nit: The debug console above, are we planning to have it forever there? 
If it's important to keep, maybe we can give more context who is writing? What about `[QuoteUpdater:useFetchMarketPrice] Fetch Market Price`

Reply:
Good point, I think we can remove it.
We already have a lot things logging in console.